.github/workflows/go.yml

```diff
@@ -180,18 +180,11 @@ jobs:
         name: windows-build-artifacts-${{ matrix.PLATFORM_ID }}
         path: build/
 
-    - run: go get github.com/x-motemen/gobump/cmd/gobump
+    - run: node _tools/parse_version.js
```

Comment:
Is there any guarantee that `node` command is installed?  I think this workflow file is for `go`.

Reply:
node is doing installed on [virtual environment](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md).